client/gulpfile.js

```diff
@@ -349,6 +345,60 @@ async function installDependenciesFromXML(xmlPath, pluginDir) {
     }
 }
 
+/**
+ * Produce plugins from fully self-contained npm packages
+ */
+async function installVisualizations(callback, forceReinstall = false) {
+    for (const pluginName of Object.keys(VISUALIZATION_PLUGINS)) {
+        const { package, version } = VISUALIZATION_PLUGINS[pluginName];
```

Comment:
Probably a non-issue, but `package` is a [reserved word in strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#keywords)

Maybe renaming it to something like this will avoid future issues.
```suggestion
        const { package: pluginPackage, version } = VISUALIZATION_PLUGINS[pluginName];
```